### PR TITLE
Derive and include a "commit_date" for new TestRun entities. Ref gh-154

### PR DESCRIPTION
**Pull request by rwaldron**
See #154 for context.

### CODE REVIEW

**Comment by rwaldron** [comment]
Rebased!

File: run/run_test.py [hunk 1]
```diff
@@ -65,40 +79,55 @@ def test_version_string_to_major_minor(self):
...
-    def test_setup_wpt_find_sha(self,
-                                mock_sha_finder,
-                                mock_patch_wpt,
-                                mock_check_call):
+    @mock.patch('subprocess.call', side_effect=stub_call)
+    @mock.patch('subprocess.check_output', side_effect=stub_check_output)
+    def test_get_commit_details_find_sha(self,
+                                         mock_sha_finder,
+                                         mock_call,
+                                         mock_check_output):
```
Comment (rwaldron) [inline run/run_test.py]: This indent is an odd quantity of spaces because `pycodestyle` insisted

¯\_(ツ)_/¯

**Comment by rwaldron** [comment]
These failures are completely effing bogus. 

https://travis-ci.org/w3c/wptdashboard/jobs/334786144

```
protoc -I../protobuf/src -I../protoc-gen-bq-schema -Iprotos \
	--python_out=run/protos protos/test_run.proto protos/token.proto protos/strawman.proto protos/browser.proto
bq_table_name.proto: File not found.
strawman.proto: Import "bq_table_name.proto" was not found or had errors.
Makefile:67: recipe for target 'py_proto' failed
make: *** [py_proto] Error 1
```

https://travis-ci.org/w3c/wptdashboard/jobs/334786145

```
protoc -I../protobuf/src -I../protoc-gen-bq-schema -Iprotos \
	--python_out=run/protos protos/test_run.proto protos/token.proto protos/strawman.proto protos/browser.proto
bq_table_name.proto: File not found.
strawman.proto: Import "bq_table_name.proto" was not found or had errors.
Makefile:67: recipe for target 'py_proto' failed
make: *** [py_proto] Error 1
```

https://travis-ci.org/w3c/wptdashboard/jobs/334786150

```
protoc -I../protobuf/src -I../protoc-gen-bq-schema -Iprotos \
	--bq-schema_out=bq-schema protos/test_run.proto protos/token.proto protos/strawman.proto protos/browser.proto
bq_table_name.proto: File not found.
strawman.proto: Import "bq_table_name.proto" was not found or had errors.
Makefile:57: recipe for target 'bq_proto' failed
make: *** [bq_proto] Error 1
```

... None of these are my problem.

**Comment by lukebjerring** [comment]
Build failures are indeed irrelevant; a Golang dep made a breaking change. See https://github.com/w3c/wptdashboard/pull/412 for build-fix.

**Pull request merged**